python/hail/expr/expression.py

```diff
@@ -87,6 +87,10 @@ def to_expr(e):
         return construct_expr(ApplyMethod('Locus', Literal('"{}"'.format(str(e)))), TLocus(e.reference_genome))
     elif isinstance(e, Interval):
         return construct_expr(ApplyMethod('Interval', Literal('"{}"'.format(str(e)))), TInterval(e.reference_genome))
+    elif isinstance(e, Call):
```

Comment:
This logic was totally missing, so Call/AltAllele couldn't be interpreted as strings